src/model/pacbuild.rs

```diff
@@ -0,0 +1,109 @@
+use chrono::NaiveDateTime as DateTime;
+use serde_derive::{Deserialize, Serialize};
+
+/// Representation of the PACBUILD file.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct PacBuild {
+    pub name: PackageId,
+    pub last_updated: DateTime,
+    pub repository: URL,
+    pub maintainer: String,
+    pub package_name: String,
+    pub description: String,
+    pub homepage: URL,
+    pub repology_version: Version,
+    pub repology: URL,
+    pub install_state: InstallState,
+    pub dependencies: Vec<PackageId>,
+    pub optional_dependencies: Vec<PackageId>,
+    pub license: String,
+    pub url: URL,
+    pub kind: Kind,
+}
+
+/// Represents a `SemVer` version.
+/// # Examples
```

Comment:
```suggestion

/// # Examples
```

Add a line gap between the description and the example for all occurences.